### ISSUE TITLE
CDN download link is outdated

### ISSUE THREAD

**Issue by aydtsang** [openai/openai-cookbook#45]
The link to download `fine_food_reviews_with_embeddings_1k.csv` in the [zero-shot classification example](https://github.com/openai/openai-cookbook/blob/838f000935d9df03e75e181cbcea2e306850794b/examples/Zero-shot_classification_with_embeddings.ipynb) is either outdated or the column headings are incorrect. It's still serving embeddings from babbage instead of ada at the moment.

Command:
```zsh
curl -s https://cdn.openai.com/API/examples/data/fine_food_reviews_with_embeddings_1k.csv | head -n1
```
Output:
```sh
ProductId,UserId,Score,Summary,Text,combined,n_tokens,babbage_similarity,babbage_search
```

**Comment by Kdr19** [comment]
curl -s https://cdn.openai.com/API/examples/data/fine_food_reviews_with_embeddings_1k.csv | head -n1

**Comment by db-aeon** [comment]
Came to post the same thing - the df is not formatted as intended for the example using ada in https://github.com/openai/openai-cookbook/blob/main/examples/Semantic_text_search_using_embeddings.ipynb

**Comment by ted-at-openai** [comment]
Will try to fix today. Thanks!

**Comment by ted-at-openai** [comment]
Fixed in #66 . Now the data is updated, and lives in the data folder of the repo.
